app/components/Channels/ChannelNodeSearch.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { withTheme } from 'styled-components'
+import { FormattedMessage, intlShape, injectIntl } from 'react-intl'
+import { Box, Flex } from 'rebass'
+import { Bar, Button, Form, Heading, Input, Panel, Text } from 'components/UI'
+import { ChannelCreateHeader } from 'components/Channels'
+import { Truncate } from 'components/Util'
+import withEllipsis from 'components/withEllipsis'
+import messages from './messages'
+
+const ClippedText = withEllipsis(Text)
+
+const SearchReasults = ({ filteredNetworkNodes }) => (
+  <Box>
+    {filteredNetworkNodes.map(node => {
+      return (
+        <React.Fragment key={node.pub_key}>
+          <Flex justifyContent="space-between" alignItems="center" py={2} pr={2}>
+            <Box mr={3}>
+              {node.alias.length > 0 ? (
+                <>
+                  <ClippedText>{node.alias.trim()}</ClippedText>
+                  <Text color="gray" fontSize="s">
+                    <Truncate text={node.pub_key} maxlen={25} />
```

Comment:
What's the difference between `ClippedText` and `Truncate` ? Sounds similar judging from the names

Reply:
`ClippedText` chips off long text with an ellipsis. `Truncate` truncates the middle of a string.